lib/reporters/base.js

```diff
@@ -152,6 +152,28 @@ exports.cursor = {
 };
 
 /**
+ * This function dumps long stack traces for exceptions having a cause()
+ * method. The error classes from
+ * [verror](https://github.com/davepacheco/node-verror) and
+ * [restify v2.0](https://github.com/mcavage/node-restify) are examples.
+ *
+ * @param {Error} ex
+ * @return {string}
+ * @api public
+ */
+
+exports.getFullErrorStack = function(err, message) {
+  var ret = err.stack || message || '';
```

Comment:
@boneskull Generally I would agree, but since the lines above and below where `getFullErrorStack()` gets called already accesses `err.message`, `err.inspect`, `err.actual` etc. I think we can safely assume that at this stage the `err` is defined and is a non-`null` value, else that code would be broken as well